### PR TITLE
Copyright change

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by maciejmrowiec** [comment]
failed license check, shouldn't we update mendertest vendor ?

File: .travis.yml [hunk 1]
```diff
@@ -106,7 +106,7 @@ script:
     # Test packages pararell (xargs -P)
     # Also build the docker container (with embedded build info), that will be pushed to dockerhub later on.
     - if [ "$JOB_TYPE" = compile_and_basic_tests ]; then
-        go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} ;
+        go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} || exit $? ;
```
Comment (maciejmrowiec) [inline .travis.yml]: ```Each command in script is processed by a special bash function. This function manipulates $? to produce logs suitable for display. Consequently, you should not rely on the value of $? in script section to alter the build behavior. See this GitHub issue for a more technical discussion.```

https://docs.travis-ci.com/user/customizing-the-build/#Note-on-%24%3F

Comment (kacf) [inline .travis.yml]: My god, are you kidding me!? Travis shell handling is really broken. But the value 1 should be good enough for our uses.

**Comment by kacf** [comment]
> failed license check, shouldn't we update mendertest vendor ?

Yep, done, and retrying.

File: .travis.yml (hunk 1 above)
Comment (maciejmrowiec) [inline .travis.yml]: It's not broken if it works as intended

Comment (maciejmrowiec) [inline .travis.yml]: this is partly issue with how yaml deals with multiline strings - it's as annoying as comfortable in some cases, if you check the same config in json it's fairly easy to spot the issue: https://travis-ci.org/mendersoftware/useradm/jobs/252799952/config

**Comment by kacf** [comment]
Cherry-picked all copyright changes to 1.1.x as well.

**Pull request merged**